src/librustc_privacy/lib.rs

```diff
@@ -939,8 +964,8 @@ impl<'a, 'tcx: 'a> TypeVisitor<'tcx> for SearchInterfaceForPrivateItemsVisitor<'
                     // free type aliases, but this isn't done yet.
                     return false;
                 }
-
```

Comment:
I think that if I don't visit the trait_ref here, it's never visited and so tests fail that expect the "private trait X in public interface" errors which are now only ever generated in `impl_trait_ref` above.